src/ReaderMiddleware.ts

```diff
@@ -587,6 +587,37 @@ export function ichainW<R2, A, O, Z, E2, B>(
     )
 }
 
+/**
+ * Indexed version of [`chainFirst`](#chainFirst).
```

Comment:
```suggestion
 * Indexed version of [`chainFirst`](#chainfirst).
```
I'm afraid otherwise anchors would not work. Same for the others.